plugins/insulter.py

```diff
@@ -15,10 +15,11 @@
 
 
 def help():
-    print config
+    print(config)
 
 def print_help():
-    print "Usage: %s person" % config["triggers"][0]
+    #print("Usage: " + config["triggers"][0] + " person")
```

Comment:
i dont think old code should be kept in comments, thats why we use git

Reply:
Missed that one, I couldn't actually find a use for print_help() if I can recall correctly.

Reply:
No sorry that was incorrect, what I could not find was the variable being called.